Client/game_sa/CBuildingSA.cpp

```diff
@@ -13,9 +13,13 @@
 #include "CBuildingSA.h"
 #include <game/CWorld.h>
 #include "CGameSA.h"
+#include "CMatrixLinkSA.h"
+#include "CDynamicPool.h"
 
 extern CGameSA* pGame;
 
+static CDynamicPool<CMatrixLinkSAInterface, PoolGrowAddStrategy<0, 500>> EXCLUSIVE_MATRIX_POOL{};
```

Comment:
```suggestion
static CDynamicPool<CMatrixLinkSAInterface, PoolGrowAddStrategy<0, 500>> g_matrixPool{};
```

Reply:
Is it right to use `g_` prefix here? This variable is local for this file.

Reply:
We don't really have a prefix for *file-global* variables, but this prefix will at least give you the signal that the variable lives in a *higher scope*.